<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QWidget" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>邮件发送</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/main.png</normaloff>:/main.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labSender">
       <property name="text">
        <string>用户名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSender">
       <property name="text">
        <string>sunkaili1999@163.com</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labSenderPwd">
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSenderPwd">
       <property name="text">
        <string>19990104xy</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labServer">
       <property name="text">
        <string>服务器:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxServer">
       <item>
        <property name="text">
         <string>smtp.163.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.126.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.qq.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smt.sina.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.sohu.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.139.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.189.com</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labPort">
       <property name="text">
        <string>端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboxPort">
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>465</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>587</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckSSL">
       <property name="text">
        <string>SSL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labSenderAddr">
       <property name="text">
        <string>发件人:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSenderAddr">
       <property name="text">
        <string>sunkaili1999@163.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labReceiverAddr">
       <property name="text">
        <string>收件人:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtReceiverAddr">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labTitle">
       <property name="text">
        <string>标  题:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTitle">
       <property name="text">
        <string>北工大校园导游——查询结果</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labAtta">
       <property name="text">
        <string>附  件:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAtta">
       <property name="text">
        <string>D:\Junior1\DS_Qt\CampusGuide_DP_QT\Email\result.jpg;D:\Junior1\DS_Qt\CampusGuide_DP_QT\Email\route.txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>浏览</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="labContent">
       <property name="text">
        <string>正  文:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txtContent">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;您所查询的路线、时间信息在截图、文件附件中。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;欢迎再次使用。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
